xio-core/build.gradle

```diff
@@ -158,3 +140,35 @@ sourceSets {
     }
   }
 }
+
+bintray_package = 'xio'
+publishing {
+  publications {
+    maven(MavenPublication) {
+      groupId project_group
+      artifactId bintray_package
+      version version
+      from components.java
+    }
+  }
+}
+
+bintray {
+  user = bintray_user //this usually comes from gradle.properties file in ~/.gradle
+  key = bintray_apikey //this usually comes from gradle.properties file in ~/.gradle
+  publications = ['maven'] //this refers to publishing.publications.maven up above ^^^
+  dryRun = bintray_dryrun.toBoolean() //Whether to run this as dry-run, without deploying
+  publish = true //If version should be auto published after an upload
+  pkg {
+    userOrg = bintray_subject
```

Comment:
should we use environment variables instead ? - `System.getenv('XXX')`

Reply:
I set this up as a project variable so if you do `./gradlew -Pbintray_dryrun=true bintrayUpload` it works as expected. My understanding with environment variables and gradle is that you need to explicitly pass them to tasks.